src/main/java/frc/robot/commands/auto/Drive10Feet.java

```diff
@@ -8,24 +8,24 @@
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.math.trajectory.Trajectory;
 import edu.wpi.first.math.trajectory.TrajectoryConfig;
 import edu.wpi.first.math.trajectory.TrajectoryGenerator;
 import frc.robot.subsystems.Drivetrain;
 
-public class Auto10Feet extends AutoSegmentAbstract {
-  public Auto10Feet(Drivetrain drivetrain, String name) {
-    super(drivetrain, name);
+public class Drive10Feet extends DriveSegmentBaseCommand {
+  public Drive10Feet(Drivetrain drivetrain) {
+    super(drivetrain, 
+        getTrajectory(
+            getDefaultTrajectoryConfig(drivetrain, true)));
   }
 
-  @Override
-  Trajectory defineTrajactory(TrajectoryConfig config) {
+  private static Trajectory getTrajectory(TrajectoryConfig config) {
     return TrajectoryGenerator.generateTrajectory(
         // Start at the origin facing the +X direction
         new Pose2d(0, 0, new Rotation2d(0)),
-        // Pass through these two interior waypoints, making an 's' curve path
-        //List.of(new Translation2d(1, 1), new Translation2d(2, -1)),
-        List.of(),
+        List.of(new Translation2d(1.5, 0)),
```

Comment:
This *should* be inconsequential but it oddly seemed to make a difference, at least when there were the other issues.